src/Form.php

```diff
@@ -447,7 +450,11 @@ public function decoratorFactory(\atk4\data\Field $f, $seed = [])
 
         $fallback_seed = ['Line'];
 
-        if ($f->type != 'boolean') {
+        if ($f->type === 'array') {
+            $limit = ($f->reference instanceof ContainsMany) ? 0 : 1;
```

Comment:
Setting the row limit to 1 is meant for ContainsOne, not ContainsMany if I am not mistaken

Reply:
> Setting the row limit to 1 is meant for ContainsOne, not ContainsMany if I am not mistaken

Because containsMany is a child of containsOne, I am testing for containsMany to set limit to 0 if true, i.e. no limit.